atom/browser/browser.h

```diff
@@ -167,6 +167,13 @@ class Browser : public WindowListObserver {
   // one from app's name.
   // The returned string managed by Browser, and should not be modified.
   PCWSTR GetAppUserModelID();
+
+  typedef HRESULT (STDAPICALLTYPE *DwmGetColorizationColor)(DWORD *, BOOL *);
```

Comment:
Compile failed without it so I had to add it in.  I didn't think it was necessary but ninja disagreed 😆